content/docs/reference/api/aut/_index.md

```diff
@@ -328,7 +240,11 @@ $ aut token balance-of --token 0xf4D9599aFd90B5038b18e3B551Bc21a97ed21c37 0x11a8
 
 ## bond
 
-Delegates an amount of Newton stake token to a designated validator. The bonded Newton amount is locked on successful processing of the method call and a bonding object for the necessary voting power change is created and tracked in memory until applied at epoch end.
+Delegates an amount of Newton stake token to a designated validator.
+On successful processing of the method call:
+
+- the bonded Newton amount is locked in the `msg.Sender`'s Newton account
+- a bonding object for the necessary voting power change is created and tracked in memory until applied at epoch end. At that block point, if the `msg.Sender` is a stake delegator account then Liquid Newton will be minted to the delegator for the bonded stake amount
```

Comment:
Note the second list item contains two sentences. So the entire list needs to be comprised of sentences:

```suggestion
Delegates an amount of Newton stake token to a designated validator.
On successful processing of the method call:

- The bonded Newton amount is locked in the `msg.Sender`'s Newton account.
- A bonding object for the necessary voting power change is created and tracked in memory until applied at epoch end. At that block point, if the `msg.Sender` is a stake delegator account then Liquid Newton will be minted to the delegator for the bonded stake amount.
```